### ISSUE TITLE
Bump base and template-haskell bounds for haskell-src-meta

### ISSUE THREAD

**Issue by athas** [haskell-infra/hackage-trustees#182]
Could a revision be added to http://hackage.haskell.org/package/haskell-src-metathat bumps the base bound from <4.12 to <4.13 and the template-haskell bound from <2.14 to <2.15? I have verified that this still builds correctly. [Upstream issue.](https://github.com/bmillwood/haskell-src-meta/issues/81)

**Comment by phadej** [comment]
Have anyone tried to contact `Ben Millwood <haskell@benmachine.co.uk>` by email about the issue?

**Comment by athas** [comment]
Apart from on GitHub?  I think not.

**Comment by phadej** [comment]
@athas, either you or @ckoparkar should sent Ben an email. Judging by his GtiHub activity he's not very active on GitHub - but still active every month. As I trustee I'll wait a little.

**Comment by ckoparkar** [comment]
Done. I sent Ben an email, and also copied you guys, and other 'haskell-src-meta' maintainers on it :)

**Comment by phadej** [comment]
FWIW, I also made https://github.com/bos/aeson/pull/675 which may or may not help you when released.

**Comment by ckoparkar** [comment]
@phadej Oops. Did you mean to post this on another thread ?

**Comment by phadej** [comment]
@ckoparkar, nope. exactly this. haskell-src-meta is blocking aeson-qq, but many people (me including) actually don't need interpolating.

**Comment by ckoparkar** [comment]
Oh, sorry. That's neat. I have a project that depends on `language-c-quote` which in turn depends on `haskell-src-meta`. Enabling GHC 8.6 on that is one more PR away :)

**Comment by ckoparkar** [comment]
We can go ahead and close this. Hackage now has the appropriate revisions.
